src/main.module.ts

```diff
@@ -24,13 +26,11 @@ import { LoggerModule } from 'nestjs-pino';
         },
       },
     }),
-    MsSqlMdmDatabaseModule,
-    MsSqlCedarDatabaseModule,
-    MsSqlCisDatabaseModule,
-    MsSqlNumberGeneratorDatabaseModule,
+    AuthModule,
+    DatabaseModule,
```

Comment:
Is there a reason we add these to `main.module` instead of `mdm.module`?

Reply:
this is updated now